app/assets/css/pages/_documentation.styl

```diff
@@ -336,3 +336,11 @@ hr.clear
                 // ideally this would be a slide out menu using Twitter Bootstrap Collapse
                 // like simple_fb_html5_sidebar
                 display: none
+
+@media only screen and (max-width: 480px)
+	.documentation
+        #content
+            article
+                #search
```

Comment:
Since the id is unique it would probably be enough to do:

```css
@media only screen and (max-width: 480px)
  #search input
    font-size: 1.5em
```